src/scss/utils/mixins.scss

```diff
@@ -31,3 +31,121 @@
     text-shadow: none;
   }
 }
+
+/**
+ * This is where all your colours are declared.
+ *
+ * The key should be equivalent to the name of the
+ * class to apply the theme colour, which is in the
+ * format of 'theme-{$element}-{$color}'.
+ *
+ * Classes should be declared on the body so that they
+ * can be accessed and applied to any element.
+ */
+$theme-colors: (
+  red: #E51016,
+  dark-red: #AD161E,
+  pink: #B32A95,
+  purple: #6239C8,
+  blue: #1F6BFE,
+  dark-blue: #123581,
+  default-accent: $brand-primary,
+  default-background: $theme-bg,
+  teal: #00837A,
+  green: #298436,
+  dark-orange: #D34300,
+  dark-ochre: #947200,
+  black: #111,
+  dark-grey: $gray,
+  light-grey: #EAEAEA,
+  white: #FFF,
+);
+
+/**
+ * Theme header color
+ */
+@mixin theme-header-color {
+  @each $name, $color in $theme-colors {
+    .theme-header-#{$name} & {
+      background-color: $color;
+    }
+  }
+}
+
+/**
+ * Theme nav color
+ */
+@mixin theme-nav-color {
+  @each $name, $color in $theme-colors {
+    .theme-nav-#{$name} & {
+      background-color: $color;
+    }
+  }
+}
+
+/**
+ * Theme footer color
+ */
+@mixin theme-footer-color {
+  @each $name, $color in $theme-colors {
+    .theme-footer-#{$name} & {
+      background-color: $color;
+    }
+  }
+}
+
+/**
+ * Theme carousel color.
+ * The background colour of the carousel when there is no image set.
+ */
+@mixin theme-carousel-color {
+  @each $name, $color in $theme-colors {
+    .theme-carousel-#{$name} & {
+      background-color: $color;
+    }
+  }
+}
+
+/**
+ * Theme accent color which defaults to background color.
```

Comment:
Instead of 'defaults to background-color'. You could use the @param docs method at the bottom of the comment, see http://sassdoc.com/annotations/#parameter